src/Avalonia.Base/PropertyStore/BindingEntry.cs

```diff
@@ -22,6 +23,8 @@ internal class BindingEntry<T> : IBindingEntry, IPriorityValueEntry<T>, IObserve
         private readonly IAvaloniaObject _owner;
         private IValueSink _sink;
         private IDisposable? _subscription;
+        private bool _isSubscribed;
```

Comment:
Probably worth packing these into an flags enum to save 4 bytes per value stored.

Reply:
If only life would be that simple 😄 Depending on the type of the entry alignment of this class will change, for example [Sharplab](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEUCuA7AHwAEAmARgFgAoa0gAgCEBLPAExYHMBRPDKATwA8AFQB8dAO4ALGLDrC6IOgGc+OMBmoBvanT10ADlCYA3AIYYYdWGdYQ8AG350A8sABWMDXQD6ECXiyANy6+kamFlZunt4+yiwA1iFU+obG5pZ0EB5eGL7KOMDKYMYGGEz2yanhGVbAEBAOvkzKAMqFxcbAMKxVYemRdPWNvsAWYFIAqgaskX16NYPCAPy+5g44MPN0oXpEAMyuOd6tEDhQYFZaHDAYQQC+1I80VKq43gBqXvvau3QHdAAImdgA4rAANbYA4GFMF0ACaUMOMNBVgAWslnrRDvQAMJ0HQpfSkEgEv6pcn6ACSeGUBlyADoABIwMwGAAUgQkjBY7Dw3F4AkELAwonZAEpxdtUjS6YyWWzOTBucw2JweHwhCiwWLJdLqbT6RpmayOVyeWr+RqhV8wPtdVK/s97kA) reducing size will do nothing since there is no free space and it will get aligned.